migrations.py

```diff
@@ -87,3 +87,26 @@ async def m005_initial(db):
         );
     """
     )
+
+
+async def m006_items(db):
+    """
+    Add items to tpos table for storing various items (JSON format)
+    Example:
+    [
+        {
+            "image": <IMG_URL>,
+            "price": <INT>,
+            "title": <STRING>,
+            "description": <STRING>,
+            "disabled": <BOOLEAN>,
+            "tax": <INT>,
+        },
+        ...
+    ]
```

Comment:
- redundant and easily to be outdated doc (`tax` not part of the model)
- the data model **is** the documentation
```suggestion
See `Item` class in models.
```